components/src/dynamo/sglang/request_handlers/handler_base.py

```
@@ -215,36 +253,65 @@ async def resume_memory_occupation(self, body: dict) -> dict:
+                if not self._memory_serving_active:
+                    try:
+                        continue_req = ContinueGenerationReqInput()
+                        await self.engine.tokenizer_manager.continue_generation(
+                            continue_req
+                        )
+                        self._memory_serving_active = True
+
+                        if self.generate_endpoint is not None:
+                            try:
+                                await self.generate_endpoint.register_endpoint_instance()
+                            except Exception as reg_err:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)